var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -345,6 +345,9 @@ def nvariant_bool(feature):
             '-DBUILD_TESTING:BOOL=OFF',
             '-DOpenGL_GL_PREFERENCE:STRING=LEGACY']
 
+        if spec.satisfies('@5.11:'):
```

Comment:
Does this include master?

Reply:
@danlipsa It does. I am building `paraview@master` nightly with this patched in and its getting it.

Reply:
Great. Than it is good to go for me. +1